.github/workflows/notebook_runner.yml

```diff
@@ -0,0 +1,22 @@
+name: Notebooks
+on:
+  pull_request:
+    branches:
+      - master
+jobs:
+  run_notebooks:
+    name: Notebook runner
+    runs-on: "ubuntu-latest"
```

Comment:
What version of python would this use?

Reply:
The default version of python that ships with the "ubuntu-latest" image, which currently is 3.8, but looks like it will be 3.10 on december 1:
https://github.com/actions/runner-images/blob/main/images/linux/Ubuntu2004-Readme.md